VideoLocker/res/menu/edit_profile.xml

```diff
@@ -2,6 +2,6 @@
 <menu xmlns:android="http://schemas.android.com/apk/res/android">
     <item
         android:id="@+id/edit_profile"
-        android:showAsAction="always"
+        android:showAsAction="always|withText"
```

Comment:
nit: I feel like the edit button is kind of small. 
<img width="474" alt="screen shot 2015-11-24 at 11 02 37 am" src="https://cloud.githubusercontent.com/assets/7101723/11372425/760d422a-929b-11e5-8301-17a507529600.png">